### ISSUE TITLE
Error requesting synonyms (french word)

### ISSUE THREAD

**Issue by Mara-Li** [phibr0/obsidian-dictionary#2]
Hello, when I try to looking for synonyms, I have nothing and error in console : 

```
VM228:2514 GET https://api.dictionaryapi.dev/api/v2/entries/fr/aux 404
eval @ VM228:2514
eval @ VM228:31
__awaiter @ VM228:27
requestSynonyms @ VM228:2511
eval @ VM228:2698
eval @ VM228:31
__awaiter @ VM228:27
requestSynonyms @ VM228:2697
eval @ VM228:3080
fulfilled @ VM228:28
Promise.then (async)
step @ VM228:30
eval @ VM228:31
__awaiter @ VM228:27
openSynonymPopover @ VM228:3058
SynonymPopover @ VM228:3050
eval @ VM228:3148
a @ app.js:1
setTimeout (async)
a @ app.js:1
setTimeout (async)
l @ app.js:1
VM228:3083 Error requesting synonyms for word aux TypeError: (intermediate value).first is not a function
    at FreeDictionarySynonymProvider.eval (eval at <anonymous> (app.js:1), <anonymous>:2519:52)
    at Generator.next (<anonymous>)
    at fulfilled (eval at <anonymous> (app.js:1), <anonymous>:28:58)
```

**Comment by phibr0** [comment]
The API we are using at the Moment unfortunetly doesn't have any synonyms for most of french words ([Look here](https://api.dictionaryapi.dev/api/v2/entries/fr/aux)). We are working on implementing more API's for more languages, until then the only language this API works well with is English.

**Comment by phibr0** [comment]
This shouldn't result in a Type Error though, i fixed this behaviour in Release 2.0.1 (243a881)
